storybook/stories/tokens/components.tsx

```diff
@@ -36,12 +36,12 @@ export const ElevationTable = ( { tokens } ) => {
 						</td>
 						<td style={ { padding: '24px' } }>
 							<div
-								aria-label={ `An square showing an example of the '${ name }' elevation styles` }
+								aria-label={ `A square showing an example of the '${ name }' ${ tokenCategory } token` }
```

Comment:
Won't really be a square for round/full, no? Should we use a more generic word like "shape"?